boot/bootutil/src/loader.c

```diff
@@ -1758,7 +1759,11 @@ boot_prepare_image_for_update(struct boot_loader_state *state,
          * if there is one.
          */
         BOOT_SWAP_TYPE(state) = BOOT_SWAP_TYPE_NONE;
-        return;
+        if (rc == BOOT_EFLASH)
+        {
+            /* Only return on error from the primary image flash */
+            return;
+        }
```

Comment:
The if-statement above uses a different coding convention

if
{
}

if {
}

Reply:
Already committed upstream.

Not worth the git paperwork to fix.